app/sentinel/internal/oidc/safe/post.go

```diff
@@ -44,17 +43,6 @@ func createAuthorizer() tlsconfig.Authorizer {
 	})
 }
 
-func decideBackingStore(backingStore string) data.BackingStore {
```

Comment:
We don’t need this anymore.

The backing store is decided at a deployment level, not on a per-secret basis.